src/packages/auth/hoc.tsx

```diff
@@ -13,8 +14,12 @@ const auth = (WrappedComponent: any) => {
 		authType: string;
 		roles: string[] | null;
 	}) => {
-		if (authType === OPEN_ID_CONNECT_AUTH)
-			return <LoginOidcComponent WrappedComponent={WrappedComponent} />;
+		if (authType === OPEN_ID_CONNECT_AUTH) {
+			const { isUserLoggedIn } = useOidc();
```

Comment:
Could you move this line before the if ? hook can not be inside condition/loop .